docs/dev_guide/ceph_s3_tests/ceph_s3_tests_guide.md

```diff
@@ -287,4 +288,46 @@ S3TEST_CONF=${PWD}/src/test/system_tests/ceph_s3_tests/test_ceph_s3_config.conf
 
 Notice that even though test commands succeeded the test itself was skipped. The test prints `1 skipped` meaning one test was skipped
 
+## Run a Single Ceph S3 Test in NC deployment Manually
+
+### NC local on your machine
+#### Account creation
+Create the needed accounts for the Ceph configurations, for example: 
+1. ceph (main account)
+2. ceph_alt (alternative account - for example I saw it used in one of the bucket policy tests)
+3. ceph_tenant  
+
+In cases that you don't need additional account you can simply copy the main account access keys in the ceph configuration file.
+
+To create the accounts in NC with access key and secret key that you chose is by running:  
+`sudo node src/cmd/manage_nsfs account add --name ceph --new_buckets_path <new-buckets-path> --access_key <access-key> --secret_key <secret-key> --uid <uid> --gid <gid>`   
+(more information can be found in [NooBaa CLI](./../../NooBaaNonContainerized/NooBaaCLI.md))  
+for example:  
+`sudo node src/cmd/manage_nsfs account add --name ceph --new_buckets_path /private/tmp/nsfs_root1 --access_key <access-key> --secret_key <secret-key> --uid 1001 --gid 1001`.  
+Note: before creating the account need to give permission to the `new_buckets_path`, in this example: `mkdir -p /private/tmp/nsfs_root1; sudo chmod 777 /private/tmp/nsfs_root1`.
+
+#### NSFS Server
+Start the NSFS server with: `sudo node src/cmd/nsfs --debug 5`  
+Note: on every code change you would need to stop the server (ctrl + c) and run it again.
+
+### Ceph S3 repository:
+1) Clone the repository [ceph/s3-test](https://github.com/ceph/s3-tests):  
+`git clone https://github.com/ceph/s3-tests.git`
+
+2) Edit the attached config, you can rename it and use this config only in the commands that you run, for example:  
+`cp s3tests.conf.SAMPLE test_nsfs_ceph_s3_config.conf`, inside the file change the following values `vi test_nsfs_ceph_s3_config.conf`:
+  - host = localhost
+  - port = 6443
+  - is_secure = True
+  - ssl_verify = False
+  - bucket prefix = your-name-{random}
+  - under "[s3 main]" under "main AWS access key" add the ceph account credentials
+  - under "[s3 alt]" under "alt AWS access key set in vstart.sh" add the ceph_alt credentials (if your tests do not need it you can copy the ceph credentials)
+  - under "[s3 tenant]" under "tenant AWS secret key set in vstart.sh" add 3. ceph_tenant credentials (if your tests do not need it you can copy the ceph credentials)
+
+3) Run a single test by running:  
+`S3TEST_CONF=<configuration-file> tox -- <test-full-name>`  
+for example:  
+`S3TEST_CONF=test_nsfs_ceph_s3_config.conf tox -- s3tests_boto3/functional/test_s3.py::test_versioning_multi_object_delete_with_marker_create`.  
```

Comment:
maybe worth mentioning that you need to run it from the s3-tests folder

Reply:
Added, I hope it is clearer now.